sormas-api/src/main/java/de/symeda/sormas/api/uuid/MismatchUuidException.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2018 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *******************************************************************************/
+package de.symeda.sormas.api.uuid;
+
+import de.symeda.sormas.api.utils.ValidationRuntimeException;
+
+@SuppressWarnings("serial")
+public class MismatchUuidException extends ValidationRuntimeException {
+	public MismatchUuidException(String entityUuid, Class entityClass, String dtoUuid) {
+		super(
+				String.format(
+						"%s dto UUID (%s) does not match entity UUID (%s)",
+						entityClass.getSimpleName(),
+						dtoUuid,
+						entityUuid));
```

Comment:
Does it just look like that, or is the formatting wrong here?

Reply:
I guess so.
@razvancornita Can you please take care of this? Make sure your IDE is configured as described here:
https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/DEVELOPMENT_ENVIRONMENT.md#step-5-configure-code-formatting-and-import-settings